stable/redmine/templates/ingress.yaml

```diff
@@ -1,23 +1,37 @@
 {{- if .Values.ingress.enabled }}
-apiVersion: {{ required "A valid .Values.networkPolicyApiVersion entry required!" .Values.networkPolicyApiVersion }}
+{{- range .Values.ingress.hosts }}
+apiVersion: extensions/v1beta1
```

Comment:
you are not using the `.Values.networkPolicyApiVersion` value any more...is that right?

Reply:
If you don't know maybe you should leave it as it was before

Reply:
.Capabilitties.ApiVersion can be used for probing what versions are supported: https://github.com/helm/helm/blob/master/docs/chart_template_guide/builtin_objects.md - the guidelines say to support the two latest minor versions of k8s.

Reply:
Great suggestion @davidkarlsen 

However, in this case I don't know if it's worth. We're just setting the **apiVersion** for an Ingress resource which is supported on **extensions/v1beta1** for every K8s version

Reply:
Done @tompizmor